lib/ansible/module_utils/redfish_utils.py

```diff
@@ -932,11 +926,19 @@ def get_nic_inventory(self, resource_type, systems_uri):
     def get_multi_nic_inventory(self, resource_type):
         ret = True
         entries = []
-        for systems_uri in self.systems_uris:
-            inventory = self.get_nic_inventory(resource_type, systems_uri)
+
+        #  Given resource_type, use the proper URI
+        if resource_type == 'Systems':
+            resource_uris = self.systems_uris
+        elif resource_type == 'Manager':
+            # put in a list to match what we're doing with systems_uris
+            resource_uris = self.manager_uri.split()
```

Comment:
Minor suggestion - I think this would be more natural than using string split().

```
            resource_uris = [self.manager_uri]
```

Reply:
Good suggestion @billdodd. I often rely on `split()` too much.